src/opcua-connector/values.ts

```diff
@@ -0,0 +1,48 @@
+import { distinctUntilChanged, map, tap } from 'rxjs/operators'
+import { OpcuaStreams, Value, Values } from './types'
+
+/** helper to transalte a value into the configured output */
+const toValue =
```

Comment:
This is something I'd write some tests for. I can imagine any number of things going wrong here...

Reply:
I opened an issue for that